models/wise-complete-policy.go

```diff
@@ -59,6 +62,44 @@ func (wisePolicy *WiseCompletePolicy) ToDomain() Policy {
 	return policy
 }
 
+func updatePolicyEndDate(policy *Policy, wisePolicy *WiseCompletePolicy) {
+	location, err := time.LoadLocation("Europe/Rome")
+	lib.CheckError(err)
+	policyEndDate := wisePolicy.Contract.PolicyExpirationDate.In(location)
+	_, offset := policyEndDate.Zone()
+	policy.EndDate = policyEndDate.Add(time.Duration(time.Second * time.Duration(offset)))
+}
```

Comment:
This probably relates to what we discussed in call earlier, but won't this code add the time offset two times?
[goplayground](https://go.dev/play/p/oYb2op9wtXx?v=goprev)

Reply:
It adds only once, but it also changes the Timezone, in the example you sent:
- The time was `2009-11-10 23:00:00 +0000 UTC`
- When adding the offset the time became `2009-11-11 00:00:00 +0000 UTC` or, in the Rome timezone `2009-11-11 01:00:00 +0100 CET`